### PR TITLE
Added ingest/reingest cattest to TGO.

### PR DESCRIPTION
**Pull request by AgoinsUSGS**
Ingests raw cassis data, spiceinit's the cube, exports it, reingests it, compares the two.

### CODE REVIEW

**Comment by AgoinsUSGS** [comment]
[PR for non-proj cat test](https://trello.com/c/KvsGKYLA/492-pr-for-non-proj-cat-test)

File: isis/src/tgo/tsts/ingestReingest/Makefile [hunk 1]
```diff
@@ -0,0 +1,24 @@
+APPNAME =
```
Comment (krlberry) [inline isis/src/tgo/tsts/ingestReingest/Makefile]: Suggestion to either use this `APPNAME` variable for `tgocassis2isis` or get rid of it. 

You don't need to change this, but I did like the approach you used in the coloredMosaic cat test you wrote that handled this as follows: 

```
INGEST = tgocassis2isis
SPICE = spiceinit
MAPFILE = mosrange
PROJECT = cam2map
MOSAIC = automos
EXPORT = isis2pds
GETKEY = getkey
CUBEIT = cubeit
```

**Comment by chrisryancombs** [comment]
Made changes according to Kristen's notes.

**Review by krlberry — APPROVED** [review]

**Comment by krlberry** [comment]
:+1: Thanks!

**Pull request merged**